src/AzureClient/Magic/ConnectMagic.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Jupyter.Core;
+using Microsoft.Quantum.IQSharp.Jupyter;
+
+namespace Microsoft.Quantum.IQSharp.AzureClient
+{
+    /// <summary>
+    ///     A magic command that can be used to connect to an Azure workspace.
+    /// </summary>
+    public class ConnectMagic : AzureClientMagicBase
+    {
+        private const string
+            ParamName_Login = "login",
```

Comment:
Not sure if this is part of the C# code conventions:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#naming-conventions
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constants

Reply:
Yes, you're right, underscore is frowned upon, I'll remove it.

---
In reply to: [426996582](https://github.com/microsoft/iqsharp/pull/144#discussion_r426996582) [](ancestors = 426996582)